code/modules/griffening_cardgame/carddatums.dm

```diff
@@ -0,0 +1,685 @@
+
+/datum/griffeningcard/equipment/incendiarygrenade
+
+	name = "Incendiary Grenade"
+	desc = "If the creature this card is equipped to attacks, any opponent creature that has less than 20 DEF is destroyed. All other opponent creatures that have 20 or higher DEF will instead have DEF lowered by 20 until the end of your turn. If this effect has concluded, destroy this card."
+	REMOVEDEF = 20
+	Attributes = list(OPPONENT_REMOVEDEF_20_ONE_TURN, DISCARD_AFTER_EFFECT, EQUIPMENT, TARGETTED, DISCARD_BELOW_DEF)
+
+/datum/griffeningcard/effect/firefightinggrenade
+
+	name = "Fire Fighting Grenade"
+	desc = "This card may be played in response to Flamethrower, Plasma Fire and Incendiary Grenade. Instantly destroy both cards. When this card is activated, the opponent's turn immediately ends."
+	Attributes = list(END_OPPONENT_TURN, NULLIFY_FIRE, DISCARD_AFTER_EFFECT, ANY_TIME)
+
+/datum/griffeningcard/equipment/plasmafire
+
+	name = "Plasma Fire"
+	desc = "This card can only be played if there's a atmospheric tech in play. When this card is played, instantly reduce the DEF of all opponent humans by 10. While this card is active, all opponent humans lose 20 DEF at the start of the opponent's turn. If a creature reaches 0 DEF due to the effects of this card, the creature is killed. If any area cards are played while Plasma Fire is active, discard Plasma Fire."
+	REMOVEDEF = 10
+	Attributes = list(GLOBAL_REMOVEDEF, REMOVEDEF_20_PER_TURN, FIRE, DISCARD_ON_AREA)
+
+/datum/griffeningcard/effect/authenticationdisk
+
+	name = "Authentication Disk"
+	desc = "This card can only be played if there's a captain in play. If the captain is on your side of the field, summon two security officer immediately on your side of the field either from your hand, deck or discard pile. If the captain is on the opponent's field, immediately summon two 'nuclear operatives' from your deck, hand or discard pile."
+	
+/datum/griffeningcard/effect/pinpointer
+
+	name = "Pinpointer"
+	desc = "When this card is played, put a 'authentication disk' card into your hand either from both yours and opponent's discard pile or deck."
+
+/datum/griffeningcard/effect/mattereater
+
+	name = "Matter Eater"
+	desc = "When this card is played, choose a face up equipment card and immediately send it to the gibbed pile and if you do, send this card to the discard pile."
+
+/datum/griffeningcard/equipment/spacesuit
+
+	name = "Space Suit"
+	desc = "Can only be equipped to humans, any human equipped with this is immune to 'hull breach', 'flamethrower', 'plasma fire' or 'incendiary grenade' as well as increase DEF by 10."
+
+/datum/griffeningcard/effect/dnaabsorbtion
+
+	name = "DNA Absorbtion"
+	desc = "Can only be played if you control a 'changeling', if you do, target a enemy creature and send it to the gibbed pile. If you do, send 'changeling' to the discard pile and immediately summon 'abomination' either from your hand, deck or discard pile."
```

Comment:
Requires changeling in your control to play. Target enemy creature card. Target is gibbed. Discard a changeling card and summon an abomination from your hand,deck, or discard pile.